diff --git a/src/case.ts b/src/case.ts
index 3c131d3..df2803d 100644
--- a/src/case.ts
+++ b/src/case.ts
@@ -1,3 +1,3 @@
 export function toKebabCase(text: string): string {
-  return text.trim().toUpperCase().split(/\s+/).join('-');
+  return text.trim().toLowerCase().split(/\s+/).join('-');
 }
diff --git a/src/truncate.ts b/src/truncate.ts
index 82e8844..86b5963 100644
--- a/src/truncate.ts
+++ b/src/truncate.ts
@@ -1,3 +1,3 @@
 export function truncate(text: string, max: number): string {
-  return text.length < max ? text : text.slice(0, max) + '…';
+  return text.length <= max ? text : text.slice(0, max) + '…';
 }
